nservicebus/recoverability/subscribing-to-error-notifications.md

```diff
@@ -24,6 +24,8 @@ snippet: SubscribeToErrorsNotifications
 
 include: notificationThread
 
+The notification instance is also injected into [dependency injection](/nservicebus/dependency-injection/).
```

Comment:
I think this sentence is still not really useful or correct. an instance isn't injected into dependency injection. An isntance might be registered with the DI container, or made available via dependency injection.

Reply:
Insert a suggestion? 😃